apps/nfw-react/src/app/project/cmp/PizzaItem/PizzaItem.cmp.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+import { Formik, Field } from 'formik';
+
+export const PizzaItem = ({ pizza, props }) => {
+  const [activeItem, setActiveItem] = useState(null);
+  const [additionalIngredientsPrice, setAdditionalIngredientsPrice] = useState(
+    0
+  );
+
+  return (
+    <li
+      className={activeItem ? 'pizza-item pizza-item--selected' : 'pizza-item'}
+    >
+      <motion.div
+        className="pizza-item__name"
+        onClick={() => setActiveItem(!activeItem)}
+      >
+        {pizza.name}
+      </motion.div>
+      <ul className="pizza-item__ingredients">
+        {pizza.ingredients.map((i) => (
+          <li className="ingredient" key={i}>
```

Comment:
(must) whole ingredient isn't best idea to be a key -> ingredient.id should be definitely better